### PR TITLE
necessary updates

### PR DESCRIPTION
**Pull request by GOPAL-YADAV-D**
Just made the updates for Hacktoberfest 2023

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,43 @@
-<p align="center"><img width="40%" src="https://hacktoberfest.digitalocean.com/assets/logo-hf19-full-10f3c000cea930c76acc1dedc516ea7118b95353220869a3051848e45ff1d656.svg"/></p>
+<p align="center"><img width="40%" src="https://hacktoberfest.com/_next/static/media/logo-hacktoberfest--horizontal.ebc5fdc8.svg"></p>
 
 # Hacktoberfest, 2019 with IEEE-VIT Student Branch
 
 SimpleMath-Flask is a Flask app that does math for you. This repository was created to help developers get started with Open Source during [Hacktoberfest](https://hacktoberfest.digitalocean.com/)
```
Comment (mintbomb27) [inline README.md]: Change the hacktoberfest URL here to https://hacktoberfest.com

File: README.md [hunk 2]
```diff
@@ -1,33 +1,43 @@
-<p align="center"><img width="40%" src="https://hacktoberfest.digitalocean.com/assets/logo-hf19-full-10f3c000cea930c76acc1dedc516ea7118b95353220869a3051848e45ff1d656.svg"/></p>
+<p align="center"><img width="40%" src="https://hacktoberfest.com/_next/static/media/logo-hacktoberfest--horizontal.ebc5fdc8.svg"></p>
 
 # Hacktoberfest, 2019 with IEEE-VIT Student Branch
```
Comment (mintbomb27) [inline README.md]: Remove 2019

File: README.md [hunk 3]
```diff
@@ -1,33 +1,43 @@
...
+Made with :heart: by [IEEE-VIT](https://ieeevit.org/)
 
 Support Open Source by participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/) :yellow_heart:
 
 ## Getting started
 
-1. To setup this project, simply run the following command:
+1.Before setting up the project,Create A virtual environment in the project directory using the following command:
+```bash
+$ python3 -m venv .venv
```
Comment (mintbomb27) [inline README.md]: Remove the `$` in this and the following commands as well.

File: README.md [hunk 4]
```diff
@@ -1,33 +1,43 @@
...
+```
+2.The environment has been created just needs to be activated:
+```bash
+$ source .venv/bin/activate  ( for mac/linux users )
```
Comment (mintbomb27) [inline README.md]: `(for mac/linux users)` and the other one maybe put that outside codeblocks?

**Review by mintbomb27 — CHANGES_REQUESTED** [review]
Please see the requested changes.

File: README.md (hunk 4 above)
Comment (GOPAL-YADAV-D) [reply README.md]: like??

**Comment by GOPAL-YADAV-D** [comment]
made the changes please have a look at it

File: README.md [hunk 5]
```diff
@@ -1,35 +1,45 @@
...
+1.Before setting up the project,Create A virtual environment in the project directory using the following command:
 ```bash
-$ pip3 install -r requirements.txt
+ python3 -m venv .venv
+```
+2.The environment has been created just needs to be activated:
+```bash
+ source .venv/bin/activate  ( for mac/linux users )  
+ source .venv/Scripts/activate ( for windows users)
+```
```
Comment (mintbomb27) [inline README.md]:
```suggestion
For mac/linux users: `source .venv/bin/activate`
For windows users: `source .venv/Scripts/activate`
```

**Comment by mintbomb27** [comment]
> made the changes please have a look at it

Resolve the issues that were fixed.

File: README.md (hunk 4 above)
Comment (mintbomb27) [reply README.md]: I've created a suggestion comment. Please check!

**Comment by GOPAL-YADAV-D** [comment]
Made the changes

**Pull request merged**